src/document.h

```diff
@@ -2024,10 +2023,21 @@ struct Document {
         }
     }
 
+    const wxChar *LoadSearch() {
+        if (Cell *c = selected.GetCell()) {
+            if (!c->text.t.Len()) return _(L"No text in this cell.");
+            sys->frame->filter->SetFocus();
+            sys->frame->filter->SetValue(c->text.t);
+            return _(L"Search string was loaded from selection.");

```

Comment:
This status is maybe superfluous, status should only be set on special events

Reply:
> This status is maybe superfluous, status should only be set on special events

Ok, I was unsure myself. I replaced it with `nullptr`.